packages/web3-eth-ens/src/config.js

```diff
@@ -27,10 +27,10 @@
  */
 var config = {
     addresses: {
-        main: "0x314159265dD8dbb310642f98f50C066173C1259b",
-        ropsten: "0x112234455c3a32fd11230c42e7bccd4a84e02010",
-        rinkeby: "0xe7410170f87102df0055eb195163a03b7f2bff4a",
-        goerli: "0x112234455C3a32FD11230C42E7Bccd4A84e02010"
+        main: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
+        ropsten: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
+        rinkeby: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
+        goerli: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```

Comment:
We can later improve this and use the address as a fallback if no custom registry address got defined.

Reply:
I've extended the list at https://github.com/ethereum/web3.js/issues/3225 with the required improvement (removing of ``config.js``).